others/valid_no_of_subarrays.cpp

```diff
@@ -0,0 +1,120 @@
+/**
+ * @file
+ * @brief - Finds Total Number of Subarrays whose net product is less than a given Number = k 
+ * [Source](https://leetcode.com/problems/subarray-product-less-than-k/)
+ * 
+ * @description :
+ * A C++ (CPP) code to find how many subarrays (contiguous part of an array) are
+ * present in an array whose product is less than a given number 'k' 
+ * e.g.- [ Input array -  arr[5]= {1, 3, 4 ,5 ,2} ] and  [ Input Number k = 5 ]
+ *  Now Subarrays of arr having product less than (k=5) are - {1} , {3} , {4} , {2} , {1 , 3}
+ * 
+ * @author - [Aryan Arya] https://github.com/hecker1002
+ * 
+ * @description :
```

Comment:
Looks like this is duplicated.

Reply:
I did not understand this query .

Reply:
There's another section that's _exactly_ the same a few lines above.

Reply:
resolved